src/CC/ShellCheck/ShellScript.hs

```diff
@@ -20,9 +22,24 @@ import           System.FilePath.Posix
 
 --------------------------------------------------------------------------------
 
+-- | List of shells the engine should be able to handle.
+validShells :: [BS.ByteString]
+validShells = ["sh", "ash", "dash", "bash", "ksh"]
+
+--------------------------------------------------------------------------------
+
+-- | List of valid shell file extensions.
+validShellExtensions :: [BS.ByteString]
+validShellExtensions = ("." <>) <$> validShells
```

Comment:
@acook I then re-used the valid shells to create a list of valid extensions to avoid us keeping the data in two places.